seminar_5/task_1.py

```diff
@@ -0,0 +1,13 @@
+"""Напишите функцию, которая принимает на вход строку — абсолютный путь до файла.
+Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла"""
+
+def parse_file_path(file_path):
+    path, filename = file_path.rsplit('/', 1)
+    name, extension = filename.split('.', 1)
+    return path, name, '.' + extension
```

Comment:
Можно сделать еще проще
```suggestion
    *path, filename = file_path.rsplit('/', 1)
    return *path, *filename.split('.')
```